File: src/main/java/com/gusrylmubarok/reddit/backend/model/RefreshToken.java
package com.gusrylmubarok.reddit.backend.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.sql.Timestamp;
import java.time.Instant;

@Entity
@Table(name = "refresh_token")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class RefreshToken {

    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    private Long id;

    @Column(name = "token")
    private String token;

    @Column(name = "created_date")
    private Instant createdDate;
}

File: src/main/kotlin/com/reddot/data/entity/RefreshToken.kt
package com.reddot.data.entity

import jakarta.persistence.*
import java.time.Instant
import java.util.*

@Entity
@Table(name = "refresh_token")
data class RefreshToken (
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long?,
    var token: String,
    var createDate: Instant,
    var createdAt: Date,
    var updatedAt: Date?
)